lib/mutant/ast/regexp/transformer/alternative.rb

```diff
@@ -0,0 +1,37 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        # Transformer for Regexp `alternative` nodes
+        #
+        # This transformer is very similar to the generic recursive mapper
+        # except for the fact that the `Regexp::Expression` class for
+        # `alternative` nodes has a unique constructor
+        class Alternative < self
+          register :regexp_sequence_expression
+
+          # Mapper from `Regexp::Expression` to `Parser::AST::Node`
+          ExpressionToAST = Class.new(Recursive::ExpressionToAST)
+
+          # Mapper from `Parser::AST::Node` to `Regexp::Expression`
+          class ASTToExpression < Transformer::ASTToExpression
+            ALTERNATIVE = IceNine.deep_freeze(
+              ::Regexp::Expression::Alternative.new(0, 0, 0)
```

Comment:
What do these `0` magic numbers actually represent? Maybe define 3 constants for them so it's more clear what their purpose is.